package.json

```diff
@@ -26,6 +26,7 @@
     "codecov:ci:changed": "lerna run codecov --since origin/main --parallel",
     "codecov:browser": "lerna run codecov:browser",
     "changelog": "lerna-changelog",
+    "lerna:link": "lerna link",
```

Comment:
Nit: is this really needed ?

Reply:
yes, used [here](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/628/files#diff-d67e2325d9922fc89305ff873c2473de9d6f739f7e3ec5aa9a029615d929c6c5R6)